config/components/mxnet.json

```diff
@@ -1,4 +1,5 @@
 {
   "name": "mxnet",
-  "cpeVendor": "apache"
+  "cpeVendor": "apache",
+  "to-be-deprecated": "20231107"
```

Comment:
This will remove `mxnet` in the next execution of the deprecation job. We are supposed to provide a month of grace. Are you sure you want to remove it right now?